### DISCUSSION TITLE
Show a reading calendar graph in user profile page

### DISCUSSION THREAD

**Discussion by nitaicharan** [dailydotdev/daily#685] [Feature Requests]
## Request
I would like to show in my profile a reading cadendar graph to show and track the days that I read or not.
It would be good to be a calendar graph like the contribuition calendar we have on GitHub.
![image](https://user-images.githubusercontent.com/14323599/186729706-8af9694d-e093-436b-badd-8800e036964e.png)

## Reason
With this new feature, the user would be able to show their knowledge level (or at least their interest level). It will also help the head hunters verify if a candidate how much or how interrested  is a candidate to acquire more knowledge in their field.

**Comment by idoshamun** [comment]
Hi @nitaicharan,

This feature already exists. Please go to your profile and enjoy your reading graph :)

**Comment by nitaicharan** [comment]
Thank you for the replay. I didn't know it
